proto/lib/config/optionsconfiguration.proto

```diff
@@ -68,3 +69,14 @@ message OptionsConfiguration {
     double          min_home_disk_free_pct = 9005 [deprecated = true];
     int32           max_concurrent_scans   = 9006 [deprecated = true];
 }
+
+message ConnectionLimits {
```

Comment:
This seems minor enough that it should live in the same file as OptionsConfiguration -- but it could be broken out into a separate file if someone insisted.